config/language-switcher.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
+    // when `false` developer should setup their own route for language switcher
+    // default: Route::any('{backpack_prefix?}/set-locale/{locale}', [\Backpack\LanguageSwitcher\Http\Controllers\LanguageSwitcherController::class, 'setLocale'])->name('language-switcher.locale')
+    'setup_routes' => true,
+
+    // when true, if the page where the picker is displayed use the `backpack route prefix` in the route, 
+    // we will also add the route prefix on the language-switcher route.
+    // eg: https://your-application.com/admin/set-locale/{locale} and not https://your-application.com/set-locale/{locale} as default
+    'use_backpack_route_prefix_on_admin_routes' => false,
```

Comment:
```suggestion
    'use_backpack_route_prefix' => false,
```

Reply:
the reason of the weird name is because theoretically it only uses the backpack prefix, if the route contains the backpack prefix. 

`use_backpack_route_prefix => true` looks to me to imply that we will ALWAYS use the route prefix, and that's not true.